week1/assignment/todo-tan/README.md

```diff
@@ -0,0 +1,13 @@
+# Todo App
+
+This is a Ruby app that lets users add/remove todo items (aka tasks) and mark any of them as completed.
+
+## User Stories
+
+As a..., I want to ..., so that I can ...
+
+
+- [ ] As a User, I want to add a new todo item
+- [ ] As a User, I want to mark an existing item as done
+- [ ] My program can load todo items from a text file
+- [ ] My program can save changes the user makes (add/remove/complete) to the file.
```

Comment:
please mark these items as DONE too. feel free to add more items and be more specific :>